tests/Cache/CacheApcStoreTest.php

```diff
@@ -51,6 +51,16 @@ public function testSetMethodProperlyCallsAPC()
         $this->assertTrue($result);
     }
 
+    public function testSetMethodWithNullTTLFails()
```

Comment:
All added test methods should be marked ` : void`

Reply:
Since we don't do this anywhere else in our test suite atm I'm going to leave this as is. 

FYI, there's currently an issue open on the ideas repo to introduce more type-hints on the framework: https://github.com/laravel/ideas/issues/1409